README.md

```diff
@@ -1,5 +1,10 @@
-Welcome! This is a public repository used to track issues, bugs and upcoming features for future releases of [Octopus Deploy](https://octopus.com). 
+# Octopus Deploy issues
 
-**For suggestions and feature ideas, please use [UserVoice](https://octopusdeploy.uservoice.com).** UserVoice allows items to be voted on, so it's much easier for us to track suggestions and prioritize accordingly. 
+Welcome! This is a public repository used to track issues, bugs and upcoming features for future releases of [Octopus Deploy](https://octopus.com).
+
+**For suggestions and feature ideas, please use [UserVoice](https://octopusdeploy.uservoice.com).** UserVoice allows items to be voted on, so it's much easier for us to track suggestions and prioritize accordingly.
```

Comment:
@MJRichardson @mcasperson Could you please confirm if this advice is still accurate, and if not, what it should be updated to? 🙇

Reply:
I'm not aware of any intention to remove UserVoice, so this is still valid.